frontend/index.html

```diff
@@ -27,18 +33,67 @@
               </a>
             </li>
           </ul>
+          -->
         </div>
       </nav>
     </div>
-    <!--News feed structure-->
+  <!--Sign up structure-->
+    <section id="register">
+      <div class="container card-container">
+          <div class="card-panel z-depth-4">
+              <a class="social-button github hoverable" id="github">
+                <img src="dist/img/github-icon.png">
+                <span class="social-button-text">Sign In With GitHub</span>
+              </a>
+              <a class="social-button linkedin hoverable" id="linkedin">
+                <img src="dist/img/linkedin-icon.png">
+                <span class="social-button-text">Sign In With Linkedin</span>
+              </a>
+              <div class="row">
```

Comment:
I think this sign-up form is not used in our service. How do you think about removing this?

Reply:
I will modify that later :0